app/controllers/contacts_controller.rb

```diff
@@ -0,0 +1,35 @@
+class ContactsController < ApplicationController
+  def new
+    @contact = Contact.new
+  end
+
+  def confirm
+    @contact = Contact.new(contact_params)
+    if @contact.invalid?
+      render :new
+    end
+
+
+  end
+
+  def back
+    @contact = Contact.new(contact_params)
+    render :new
+  end
+
+  def create
+    @contact = Contact.new(contact_params)
+    if @contact.save
+      ContactMailer.sendmail_user(@contact).deliver
+      redirect_to "/blogs", notice: "お問い合わせありがとうございます！"
```

Comment:
```suggestion
      redirect_to blogs_url, notice: 'お問い合わせありがとうございます！'
```
名前付きルートを使った方がいいです。
blogs_controllerのnoticeをシングルクォーテーションにするならば、こちらも合わせた方がいいです。